src/sap.ui.core/test/sap/ui/core/qunit/BindingParser.qunit.html

```diff
@@ -659,6 +659,51 @@
 		oModel.setProperty("/blue", true);
 		strictEqual(oIcon.getColor(), "*red*", "one time binding -> value unchanged");
 	});
+	
```

Comment:
You did not Change the BindingParser itself, thus I find it strange to Change its test. What is the reason behind this? Maybe we can find a better place for These Tests.